templates/settings.py.erb

```diff
@@ -27,3 +27,11 @@ WORKING_DIRECTORY = "<%= scope['pulpcore::cache_dir'] %>"
 <% if scope['pulpcore::remote_user_environ_name'] -%>
 REMOTE_USER_ENVIRON_NAME = '<%= scope['pulpcore::remote_user_environ_name'] %>'
 <% end -%>
+AUTHENTICATION_BACKENDS = ['pulpcore.app.authentication.PulpNoCreateRemoteUserBackend']
+
+REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES = (
+    'rest_framework.authentication.SessionAuthentication',
+    'pulpcore.app.authentication.PulpRemoteUserAuthentication'
+)
+
+ALLOWED_IMPORT_PATHS = ['/var/www/test_repos']
```

Comment:
What is this line handling?

Reply:
Pulp 3 added a restriction on what paths you can sync from using the 'file://' 'protocol'.    So this is needed to allow our tests to run, however @sjha4 i think we probably want to set this to a more generic path for users and probably one that would allow python to optimize the copy (which later versions of python do).  I would recommend /var/lib/pulp/sync_imports/  or something like that.  And we'll need to update our tests to use that

Reply:
Yes..this is to handle our tests only for now..Let me try modifying tests with this /var/lib/pulp/sync_imports/ and check if I run into any issues.. :+1:

Reply:
If only for testing, a comment in this case would be useful as that is information I'm sure will be lost on us down the road.

Reply:
Added the comment and updated katello tests to use path '/var/lib/pulp/sync_imports/'

Reply:
@ehelms its not *only* for tests, pulp will ONLY sync from file:// repos in the directories we specify, this is a change from pulp2.  Our tests that do this need to use this directory as well.